static/sass/components/_hero.scss

```diff
@@ -25,3 +25,54 @@
     }
   }
 }
+
+// Hero Images
+// ===========
+
+
+.hero {
+  max-height: 35vh;
+  overflow: hidden;
+  position: relative;
+
+  @include above('medium-page') {
+    max-height: 45vh;
+  }
+
+  + .title {
```

Comment:
yes, that is the plan once we get template grid in here. I was waiting for template grid but I suppose we don't have to.